### ISSUE TITLE
CVE-2022-21724 vulnerability related to postgresql dependency

### ISSUE THREAD

**Issue by Painyjames** [keycloak/keycloak#10763]
### Describe the bug

AWS Inspector has found a vulnerability related to the postgresql driver dependency.
The details of said vulnerability can be found [here](https://nvd.nist.gov/vuln/detail/CVE-2022-21724#vulnConfigurationsArea) and can be sorted out if [that dependency](https://github.com/keycloak/keycloak/blob/main/pom.xml#L152) is not between this versions 42.3.0 and 42.3.2 or before 42.2.25.

### Version

1.6.1

**Comment by DGuhr** [comment]
@stianst this has the area/dist/quarkus label, but we use 42.3.3 - so for quarkus it should be solved. Not sure how to proceed :)

**Comment by abstractj** [comment]
Indeed, the Quarkus distribution is not affected, only the Keycloak legacy.

**Comment by abstractj** [comment]
@stianst interesting enough, Snyk, Dependabot and depscan could not catch it https://snyk.io/vuln/maven:org.postgresql%3Apostgresql

**Comment by Painyjames** [comment]
If it helps, this is what we are getting from AWS Inspector, which I believe uses Clair to scan for vulnerabilities.
<img width="462" alt="Screenshot 2022-03-21 at 11 32 25" src="https://user-images.githubusercontent.com/1241373/159253055-99592959-6b5a-46a9-a106-7676d2bb4384.png">

**Comment by stianst** [comment]
We don't bundle the PostgreSQL JDBC driver in the legacy WildFly distribution, only in the legacy container

**Comment by Painyjames** [comment]
That's correct, so I guess the fault might be there?

**Comment by abstractj** [comment]
@stianst out of curiosity, why you decided to reopen?
